qiskit/result/utils.py

```diff
@@ -14,18 +14,24 @@
 
 from functools import reduce
 from re import match
+from copy import deepcopy
 
 from qiskit.exceptions import QiskitError
+from qiskit.result.result import Result
+from qiskit.result.postprocess import _bin_to_hex
 
 
-def marginal_counts(result, indices=None):
+def marginal_counts(result, indices=None, in_place=False):
```

Comment:
```suggestion
def marginal_counts(result, indices=None, inplace=False):
```

Other uses of this parameter have this form.

Reply:
Sure! Changing that and the tests as well.